api/src/app.module.ts

```diff
@@ -22,7 +22,7 @@ import { UserEntity } from './user/entities/user.entity';
   imports: [
     ConfigModule.forRoot(),
     LoggerModule.forRoot({
-      pinoHttp: { prettyPrint: { colorize: true, singleLine: process.env.NODE_ENV === 'production' } },
+      pinoHttp: { prettyPrint: { colorize: true, singleLine: true } },
```

Comment:
I feel `singleline` is much easier to read, so I think I just keep it on any env.
This was implemented by me earlier. 
![image](https://user-images.githubusercontent.com/46575401/118187779-48d14b00-b3f4-11eb-8579-1eeab5370fae.png)